packages/wonder-blocks-dropdown/components/dropdown-core-virtualized.js

```diff
@@ -108,11 +103,7 @@ class DropdownCoreVirtualized extends React.Component<Props, State> {
         // after the non-virtualized items are rendered, we get the container
         //  width to pass it to react-window's List
         if (rootNode) {
-            const clientWidth = rootNode.getBoundingClientRect().width;
-            const width =
-                clientWidth < MAX_ALLOWED_WIDTH
-                    ? clientWidth
-                    : MAX_ALLOWED_WIDTH;
+            const width = rootNode.getBoundingClientRect().width;
```

Comment:
If we wanted to avoid a call to `getBoundingclientRect()` for perf reasons we could set the width to be the width of the window since it's getting clipped by the dropdown anyways.  This is fine too.